control/src/main.rs

```diff
@@ -0,0 +1,162 @@
+mod runtime;
+mod commands;
+mod helpers;
+
+use codec::Encode;
+use clap::{Parser, Subcommand, ValueEnum};
+use subxt::{OnlineClient, PolkadotConfig};
+use std::io::Write;
+use alloy_primitives::{Address, Bytes, FixedBytes};
+
+#[derive(Debug, Parser)]
+#[command(name = "snowbridge-control", version, about, long_about = None)]
+struct Cli {
+    /// Output format of preimage
+    #[arg(value_enum, default_value_t=Format::Hex)]
+    format: Format,
+
+    #[command(subcommand)]
+    command: Command,
+}
+
+#[derive(Debug, Subcommand)]
+pub enum Command {
+    /// Change the gateway operating mode
```

Comment:
We should maybe have a command for the initial deploy that creates a single preimage for the whole deploy like we would do in production, and then we consume the command from start services in order to set up the local testnet.

This would be the most useful command given the kusama deployment is soon.

Reply:
Yeah definitely! The main helper function `wrap_calls` can already emit an XCM script that contains multiple Transacts (if we go that route instead of `Utility.batch`).

Reply:
Seems `Utility.batch(_all)` is not allowed in BH runtime through xcm so this one is super useful.  The caveat is to be careful to not exceed the weight limit for xcm.
https://github.com/polkadot-fellows/runtimes/blob/2ef38e75b43b2b91d160936843fa8b3e4ffbf5fc/system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs#L291C20-L291C29